mix.exs

```diff
@@ -29,23 +29,23 @@ defmodule Bandwidth.Mixfile do
 
   defp package do
     [ files: ["lib", "mix.exs", "README.md", "LICENSE.md"],
-      contributors: ["Tyler Cross"],
+      contributors: ["Tyler Cross", "Ross Lonstein"],
       licenses: ["MIT"],
       links: %{
-        GitHub: "https://github.com/wtcross/elixir-bandwidth",
+        GitHub: "https://github.com/rlonstein/elixir-bandwidth",
```

Comment:
Pardon my ignorance, is there any reason that this _shouldn't_ be `https://github.com/Bandwidth/elixir-bandwidth/` ? I'm not too hip on elixir package management, so I could be way off base.

Regardless, thank you so much for the update!

Reply:
No reason, was trying to get the coveralls.io working and swapped out the original author's repo for my own. I'll switch that to your upstream one.

Reply:
gotcha! Thank you for the feedback.

Reply:
Stripped out coveralls.